src/layers/legacy/medCoreLegacy/data/medMetaDataKeys.cpp

```diff
@@ -34,7 +34,7 @@ namespace medMetaDataKeys {
 
     // STUDY
     MEDCORELEGACY_EXPORT const Key StudyID("StudyID", "Study ID");
-    MEDCORELEGACY_EXPORT const Key StudyDicomID("StudyDicomID", "Study Dicom ID");
+    MEDCORELEGACY_EXPORT const Key StudyInstanceUID("StudyInstanceUID", "Study Instance UID");
```

Comment:
by the way, are these changes interesting for the original medInria also?

Reply:
StudyDicomID is not a valid DICOM tag, [StudyInstanceUID](https://dicom.innolitics.com/ciods/ct-image/general-study/0020000d) is the correct one.
Yes, it should be propagated to medInria